docs/Using-Tensorboard.md

```diff
@@ -15,11 +15,15 @@ start TensorBoard:
 3. From the command line run :
 
       ```sh
-      tensorboard --logdir=summaries
+      tensorboard --logdir=summaries --port=6006
       ```
 
 4. Open a browser window and navigate to [localhost:6006](http://localhost:6006).
 
+**Note:** The default port tensorboard uses is 6006. If there is an existing session
```

Comment:
TensorBoard, not tensorboard. Please do a search/replace.

Reply:
👍